concurrency-limits-core/src/test/java/com/netflix/concurrency/limits/limiter/SimpleLimiterTest.java

```diff
@@ -82,4 +90,63 @@ public void testSimpleBypassLimiterDefault() {
         Assert.assertFalse(limiter.acquire("admin").isPresent());
     }
 
+    @Test
+    public void testConcurrentSimple() throws InterruptedException {
+        final int THREAD_COUNT = 100;
+        final int ITERATIONS = 1000;
+        final int LIMIT = 10;
+
+        SimpleLimiter<String> limiter = (SimpleLimiter<String>) TestPartitionedLimiter.newBuilder()
+                .limit(FixedLimit.of(LIMIT))
+                .partition("default", 1.0)
+                .build();
+
+        ExecutorService executor = Executors.newFixedThreadPool(THREAD_COUNT);
+        CountDownLatch startLatch = new CountDownLatch(1);
+        CountDownLatch endLatch = new CountDownLatch(THREAD_COUNT);
+        AtomicInteger successCount = new AtomicInteger(0);
+        AtomicInteger rejectionCount = new AtomicInteger(0);
+        AtomicInteger maxConcurrent = new AtomicInteger(0);
+
+        for (int i = 0; i < THREAD_COUNT; i++) {
+            executor.submit(() -> {
+                try {
+                    startLatch.await();
+                    for (int j = 0; j < ITERATIONS; j++) {
+                        Optional<Listener> listener = limiter.acquire("default");
+                        if (listener.isPresent()) {
+                            try {
+                                int current = limiter.getInflight();
+                                maxConcurrent.updateAndGet(max -> Math.max(max, current));
+                                successCount.incrementAndGet();
+                                Thread.sleep(1); // Simulate some work
+                            } finally {
+                                listener.get().onSuccess();
+                            }
+                        } else {
+                            rejectionCount.incrementAndGet();
+                        }
+                    }
+                } catch (InterruptedException e) {
+                    Thread.currentThread().interrupt();
+                } finally {
+                    endLatch.countDown();
+                }
+            });
+        }
+
+        startLatch.countDown();
+        endLatch.await();
+        executor.shutdown();
+        executor.awaitTermination(10, TimeUnit.SECONDS);
+
+        System.out.println("Success count: " + successCount.get());
+        System.out.println("Rejection count: " + rejectionCount.get());
+        System.out.println("Max concurrent: " + maxConcurrent.get());
```

Comment:
same re: avoiding System.out in tests

Reply:
fixed in [09a1e6f](https://github.com/Netflix/concurrency-limits/pull/204/commits/09a1e6fd1d649b881c067d18f294011f2360cbe5)